pkg/bolttools/client.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2016 Mirantis
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bolttools
+
+import (
+	"github.com/boltdb/bolt"
+)
+
+type BoltClient struct {
+	db *bolt.DB
+}
+
+func NewBoltClient() (*BoltClient, error) {
+	db, err := bolt.Open("virtlet.db", 0666, nil)
```

Comment:
IMO path to file should be absolute to have more control where it will be created if virtlet will be started for example from systemd service file (this will remove need to set `WorkingDirectory`). It's not necessary - i'm pointing this only to discuss - as mentioned `WorkingDirectory` or simple `cd` in script in other init system will do the work.

Also do we need so beauty file mode? IMO `0600` should be enough.